lib/pavilion/plugins/commands/view.py

```diff
@@ -68,16 +68,19 @@ def run(self, pav_cfg, args):
         self.logger.debug("Finding Configs")
 
         sys_vars = system_variables.get_vars(True)
-
-        configs = self._get_tests(
-            pav_cfg=pav_cfg,
-            host=args.host,
-            test_files=[],
-            tests=tests,
-            modes=args.modes,
-            overrides=overrides,
-            sys_vars=sys_vars,
-        )
+        try:
+            configs = self._get_tests(
+                pav_cfg=pav_cfg,
+                host=args.host,
+                test_files=[],
+                tests=tests,
+                modes=args.modes,
+                overrides=overrides,
+                sys_vars=sys_vars,
+            )
+        except Exception as err:  # pylint: disable=W0703
+            fprint(err, file=self.errfile, color=31)
+            sys.exit(1)
```

Comment:
- Return an error code, don't exit (the base pavilion script will handle exiting properly). 
 - Print a useful error, not just the exception.
 - Use the utils.COLOR constants, not just numbers. 
 - Figure out what exceptions could be raised, and catch those. Don't catch generic exceptions.